README.md

```diff
@@ -129,7 +168,16 @@ SQL database. If you need an example, you can use `people` and `wishes` models.
 Your answer...
 
 ```
-Replace this with your answer
+A schema sets the contents of the rows and columns of a table in a database.
+
+class People
```

Comment:
This part of the question was looking for an english description of how relationships are represented in SQL. Something like how the `wishes` table might have a column for `person_id` in order to relate that table with the `people` table in a relationship of many `wishes` to one `person`